src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -1,10 +1,15 @@
 ---
 title: 'Test life cycle'
-excerpt: 'The four distinct life cycle stages in a k6 test are "init", "setup", "VU" and "teardown".'
+excerpt: 'The four distinct life cycle stages in a k6 test are "init", "setup", "VU", and "teardown".'
 ---
 
-The four distinct life cycle stages in a k6 test are "init", "setup", "VU" and "teardown"
-Throughout the documentation, you will also see us referring to it as "init code", "VU code" etc.
+A k6 test has four distinct stages, which always run in the same order:
+
+1.  *Init code* initializes VUs
+2. *Setup code* sets up data and the test environment
+3. *VU* code runs the test
+4. *Teardown* cleans up data and stops the test environment
+
```

Comment:
```suggestion
1. *Init code* initializes VUs
2. *Setup code* sets up data and the test environment
3. *VU* code runs the test
4. *Teardown* cleans up data and stops the test environment

```

If we are listing `setup` and `teardown` I guess we should list `handleSummary`

Also, we do have `code` three times, two times bold, one not. Also, we could say `teardown code` the same we can say `setup code` so, which all seems a bit inconsistent.

Reply:
Thanks! I fixed the list.

I'm not sure about whether to add `handleSummary` here. While it's definitely part of the code, it doesn't seem as much a part of the test's "life cycle". These four stages distinctly influence how the test runs. `handleSummary` influences output, but not the actual test.

Does that make sense to you? This is just my initial opinion here.